instapi/user_data.json

```diff
@@ -0,0 +1 @@
+{"username_id": 42662604294, "rank_token": "42662604294_c619e48c-9c53-460c-b065-2c6e9c99b032", "csrftoken": "y1OZrn8UsRs9vRPSMnJB9KOLPQgPis0O", "ds_user": "subd_sequrity", "ds_user_id": "42662604294", "rur": "ATN", "sessionid": "42662604294%3AqQ5j3tjVsN9fAT%3A22", "md5": "58e49e3370507ce31940658e9eeadfe2"}
```

Comment:
I suggest to rename this file to `session.json`

Reply:
Also I suggest to add this file to .gitignore

Reply:
> I suggest to rename this file to `session.json`

Sorry, but I can't agree, it's not just session data

Reply:
> Also I suggest to add this file to .gitignore

Yea, nice idea, thx